Mapper/BaseGroupedMapper.php

```diff
@@ -70,11 +70,17 @@ public function with($name, array $options = array())
             'collapsed' => false,
             'class' => false,
             'description' => false,
+            'label' => $this->admin->getLabelTranslatorStrategy()->getLabel($name, $this->getName(), 'group'),
             'translation_domain' => null,
             'name' => $name,
             'box_class' => 'box box-primary',
         );
 
+        // BC layer for manual label
+        if (strpos($name, '.')) {
+            $defaultOptions['label'] = $name;
+        }
```

Comment:
> This is the same thing than using setter injection, no ?

Not really since we won't have to remove a setter. Plus you can check whether the argument is provided or not in the constructor, and throw an exception if it isn't.